cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/requestrate/AtomicCounter.java

```diff
@@ -0,0 +1,35 @@
+package io.smartcat.cassandra.diagnostics.module.requestrate;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * Atomic counter implementation.
+ */
+public class AtomicCounter {
```

Comment:
'Atomic' is an implementation detail so I think a name like 'Counter' is quite enough.